### ISSUE TITLE
Readd Windows builds

### ISSUE THREAD

**Issue by jakirkham** [conda-forge/datafusion-feedstock#13]
In PR ( https://github.com/conda-forge/datafusion-feedstock/pull/10 ), Windows builds were disabled. Would be good to reenable them.

**Comment by charlesbluca** [comment]
Seems like as of datafusion 28.0.0, we get some LLVM failures during compilation on Windows:

```
  LLVM ERROR: out of memory
  Allocation failed
  error: could not compile `datafusion-python` (lib)
```

I had intended to resolve this issue with #27, but in the interest of getting out a new conda-forge release might need to push a fix for this into a follow up PR

**Comment by charlesbluca** [comment]
This should now be resolved with #34

**Comment by jakirkham** [comment]
Thanks Charles! 🙏
